src/main/java/com/example/springboot/SpotifyConfig.java

```diff
@@ -0,0 +1,28 @@
+package com.example.springboot;
+
+import org.springframework.beans.factory.annotation.Value; 
+import org.springframework.context.annotation.Configuration; 
+
+@Configuration
+public class SpotifyConfig {
+    @Value("${spotify.clientId}")
```

Comment:
The values come from the `application.properties` file. Just don't want to add API tokens to Git where they're publicly visible.